src/Symlex/Router/TwigRouter.php

```diff
@@ -99,11 +99,12 @@ public function route($routePrefix = '', $servicePrefix = 'controller.web.', $se
             return $response;
         };
 
-        $indexRequestHandler = function (Request $request) use ($app, $container, $servicePrefix, $servicePostfix, $webRequestHandler) {
-            return $webRequestHandler('index', $request, 'index');
+        $indexRequestHandler = function (Request $request, $action='index') use ($app, $container, $servicePrefix, $servicePostfix, $webRequestHandler) {
+            return $webRequestHandler('index', $request, $action);
         };
 
         $app->get($routePrefix . '/', $indexRequestHandler);
+        $app->get($routePrefix . '/{action}', $indexRequestHandler)->assert('action', '.+');
```

Comment:
Is to have more than one action in the IndexController. if IndexController have index and edit actions the routes "/" and "/edit" will be catched by IndexController and the other routes with index actions by their respectives controllers